routes/trainer_routes.go

```diff
@@ -0,0 +1,12 @@
+
+package routes
+
+import (
+	"trainder-api/controllers"
+
+	"github.com/gin-gonic/gin"
+)
+
+func TrainerRoute(router *gin.Engine) {
+	router.GET("/trainer", controllers.GetTrainer())
+}
```

Comment:
Make this a protected route also, only logged in user should be allow to find trainer right?